doc/user-guide/add_to_user_guide.md

```diff
@@ -0,0 +1,11 @@
+# Unrecognized and Unsupported Data Types
```

Comment:
First - the file's name doesn't tell me what's inside. Second - I can't see any link to this file, nobody will ever reach it without a link...